frontend/src/components/module-editor/SideBarModuleOverview.tsx

```diff
@@ -22,16 +28,34 @@ const SideBarModuleOverview = (): React.ReactElement => {
   const setFocus = (index: number) =>
     dispatch({ type: "set-focus", value: module.lessons[index] });
 
-  const createLesson = (title: string) =>
-    dispatch({
-      type: "create-lesson",
-      value: {
-        course: courseID,
-        module: course.modules[moduleID].id,
-        title,
-        content: [],
-      },
-    });
+  const resetState = () => {
+    setTitle("");
+    setErrorMessage("");
+    setIsInvalid(false);
+  };
+
+  const createLesson = (lessonTitle: string) => {
+    if (title) {
+      dispatch({
+        type: "create-lesson",
+        value: {
+          course: courseID,
+          module: course.modules[moduleID].id,
+          title: lessonTitle,
+          content: [],
+        },
+      });
+      resetState();
+      onClose();
```

Comment:
Do we want similar naming conventions for onClose as we have right now with openCreateLessonModal? (ie. closeCreateLessonModal). We could do that for isOpen too with isCreateLessonModalOpen. This is under the assumption you did this for clarity of what the disclosure is corresponding to!

Reply:
> Great job! One small question: part of the figma for this feature has a max character count for the field. Did you have any plans to implement that? We don't have to do it in this PR but if not, let me know so I can create a separate issue for it!

@jfdoming Would you be able to link the figma here? I can't seem to find where it states the the max character count for the field. But for now, I'm thinking maybe it would be good to create a new issue as an improvement feature.

Reply:
Sure, I'll split this off as an issue in the backlog. Figma link is https://www.figma.com/file/Dimm13cjrpYU5LkcrV6VxF/Course-Editor?node-id=703%3A5903

Reply:
@mslwang thanks for your input! i actually got rid of the initial openCreateLessonModal naming entirely since it didn't seem to make sense to add more complexity than necessary. It'd be great if you could review the new changes again!